internal/configure/common/item_set.go

```diff
@@ -35,6 +35,7 @@ const (
 	REQUIRE_BOOL
 	REQUIRE_INT
 	REQUIRE_POSITIVE_INTEGER
+	REQUIRE_SLICE
```

Comment:
Now we add the require type `REQUIRE_SLICE`, but there is no corresponding implementation for it.

Reply:
Done.